app/assets/stylesheets/jquery-ui/custom-theme/jquery-ui-1.8.13.custom.scss

```diff
@@ -59,26 +59,26 @@
 .ui-widget { font-family: Arial,Helvetica,sans-serif; font-size: 1em; }
 .ui-widget .ui-widget { font-size: 1em; }
 .ui-widget input, .ui-widget select, .ui-widget textarea, .ui-widget button { font-family: Arial,Helvetica,sans-serif; font-size: 1em; }
-.ui-widget-content { border: 1px solid #aaaaaa; background: #ffffff image_path('ui-bg_flat_75_ffffff_40x100.png') 50% 50% repeat-x; color: #222222; }
+.ui-widget-content { border: 1px solid #aaaaaa; background: #ffffff ; color: #222222; }
```

Comment:
Color literals like `#aaaaaa` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#222222` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background, border, color<br>Declaration should not have a space before the terminating semicolon